element_array_ephys/plotting/widget.py

```diff
@@ -1,30 +1,70 @@
-import ipywidgets
+import ipywidgets as widgets
 from .. import ephys_report
 import matplotlib.image as mpimg
 import matplotlib.pyplot as plt
 import pathlib
 import typing as T
 from IPython.display import display
+from .. import ephys_no_curation as ephys
```

Comment:
How would you know if it is the `ephys_no_curation` that is being activated and used here?

Reply:
changed so that ephys_widget inputs the activate ephys ephys module as an argument